eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version4/ChannelCodecs4.scala

```diff
@@ -564,16 +602,31 @@ private[channel] object ChannelCodecs4 {
     }
 
     val commitmentsCodecWithoutFirstRemoteCommitIndex: Codec[Commitments] = (
-      ("params" | paramsCodec) ::
+      ("params" | paramsCodec) >>:~ { params =>
```

Comment:
This is where we copy the `commitmentFormat` from `ChannelParams` to `Commitment`, which will be needed at some point I think.